.github/workflows/deploy_template.yaml

```diff
@@ -0,0 +1,38 @@
+name: Deploy Template
+
+on:
+  workflow_call:
+    inputs:
+      source_branch:
+        type: string
+        required: true
+        description: Source Branch
+      target_branch:
+        type: string
+        required: true
+        description: Target Branch
+jobs:
+  deploy:
+    name: Deploy ${{ inputs.target_branch }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/create-github-app-token@v1
+        name: Generate GitHub Token
+        id: app-token
+        with:
+          app-id: ${{ secrets.LUNATECH_BLOG_APP_ID }}
+          private-key: ${{ secrets.LUNATECH_BLOG_APP_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ steps.app-token.outputs.token }}
+          fetch-depth: '0'
+
+      - name: Merge from ${{ inputs.source_branch }} to ${{ inputs.target_branch }}
+        env:
+          GITHUB_TOKEN: ${{ steps.app-token.outputs.token }}
```

Comment:
I dont think you need this anymore, the authentication has already been setup in before

Reply:
Right, I had to use it in a similar pipeline to create PRs 🙈